runner.sh

```diff
@@ -1,30 +1,40 @@
 #!/bin/bash
+
 set -e
 
 sudo touch requirements.txt
 user=nsls2
-image="condaforge"
+image="nsls2-analysis-2021-1.2"
 timestamp=$(date +%Y%m%d%H%M%S)
 env_name="nsls2-analysis-2021-1.2"
-#docker build . -t conda --target build
-#docker build . -t temp-image --target export
-#docker image build . -t $user/${image}:latest \
- #                  -t $user/${image}:${timestamp}
-#docker run -it --rm -v $PWD:/build $user/${image}:${timestamp} bash -l /build/export.sh
-#docker create -ti --name temp-container temp-image bash
-#docker cp temp-container:/build .
+
+if [ -e nsls2-analysis-2021-1.2.tar.gz ]
+then rm nsls2-analysis-2021-1.2.tar.gz
+     rm nsls2-analysis-2021-1.2.yml
+     rm nsls2-analysis-2021-1.2-sha256sum.txt
+fi
+
+docker image build . -t $user/${image}:latest \
+                     -t $user/${image}:${timestamp}
+docker run -it --rm -v $PWD:/build $user/${image}:${timestamp} bash -l /build/export.sh
 sudo chown -v $USER: ${env_name}.tar.gz ${env_name}-sha256sum.txt ${env_name}.yml
-exit 1
-
-echo $QUAY_PASS > ~/password.txt
-cat ~/password.txt | docker login quay.io --username $QUAY_USER --password-stdin
-docker commit temp quay.io/$QUAY_REPOS
-docker push quay.io/$QUAY_REPOS
-docker rm -f temp-container
-docker image rm $user/${image}:latest
-#docker image rm temp-image
-
-#user=nsls2
-#image="condaforge"
-#timestamp=$(date +%Y%m%d%H%M%S)
-#docker image build . -t $user/${image}:latest \
+
+latest_holder="$user/${image}:latest"
+timestamp_holder="$user/${image}:${timestamp}"
+export latest_holder
+export timestamp_holder
+
+## Push to Quay
+#chmod +x push_quay.sh
+#./push_quay.sh
+
+## Push to Dockerhub
+#chmod +x push_dockerhub.sh
+#./push_dockerhub.sh
+
+## Push to GHCR
+#chmod +x push_ghcr.sh
+#./push_ghcr.sh
```

Comment:
This commented block can be conditionally created via the jinja template.

Reply:
Done. I will look over this file again tomorrow since I'm tired now and may have made some mistakes.

Leaving unresolved.